laurent22/joplin
# Mobile: Configurable editor font size

# Summary
- It has been requested in several places on the forum that the mobile editor's font size be configurable (e.g. https://discourse.joplinapp.org/t/ios-font-size/11061/17).

# Testing plan
- Open the plain text editor, observe the font size.
- Close the editor, open settings, adjust the editor's font size.
- Open the plain text editor again, observe the font size.
- Open the CodeMirror text editor, observe the font size.
- Close the editor, open settings, adjust the editor's font size.
- Open the CodeMirror text editor again, observe the font size.

# Notes
- At present, this has only been tested on Android.
